pyproject.toml

```diff
@@ -0,0 +1,95 @@
+[build-system]
+requires      = ["setuptools>=61.0.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "actinia-core"
+version = "4.7.1"
+description = "Actinia Core is an open source REST API for scalable, distributed, high performance processing of geographical data that uses mainly GRASS GIS for computational tasks"
+readme = "README.md"
+authors = [
```

Comment:
Shall we make this list match the list of authors in https://github.com/actinia-org/actinia-core/blob/main/CITATION.cff
?

Reply:
done